shell/common/animator.cc

```diff
@@ -133,7 +134,8 @@ void Animator::BeginFrame(fml::TimePoint frame_start_time,
   // to service potential frame.
   FML_DCHECK(producer_continuation_);
 
-  last_frame_begin_time_ = frame_start_time;
+  last_frame_begin_time_ = fml::TimePoint::Now();
```

Comment:
when we post a trace event for `VsyncSchedulingOverhead` we also use `fml::TimePoint::Now()`. Can we unify both these and maybe post a trace event here instead?

Reply:
Not 100% sure I understand but I made a commit moving the TaceEventAsyncComplete line. PTAL.